contrib/automation_tests/orbit_manual_instrumentation.py

```diff
@@ -62,7 +62,10 @@ def main(argv):
                      expected_minimum_track_count=3,
                      expected_maximum_track_count=4),
         ToggleCollapsedStateOfAllTracks(),
-        CheckTimers(track_name_filter="OrbitThread_*")
+        # For the same reason explained above 'FilterTracks' we only check if at leat one track
+        # contains timers; strictly speaking there are three tracks with timers and there might
+        # be anotherone without timers.
```

Comment:
> anotherone
Also, I remember, you've argued there might be multiple "ghost" threads, no?

Reply:
Yeah, the current assumption in the code is that there is only one (`expected_maximum_track_count=4` above). This would keep working of course with an arbitrary number of extra threads.